scripts/build.ps1

```diff
@@ -29,9 +29,9 @@ if (!$env:DOTNET_BUILD_VERSION) {
     $buildnumber = $commitTime.Days
     $revnumber = $commitTime.TotalSeconds
     
-    $VersionSuffix = "$buildnumber.$revnumber"
+    $VersionSuffix = "dev-$revnumber"
```

Comment:
I get that the format must be different, but why are we diverging versioning schemes between packages and binaries?